sources/engine/Stride.Games/GameTime.cs

```diff
@@ -94,10 +97,42 @@ public GameTime(TimeSpan totalTime, TimeSpan elapsedTime)
         /// <value><c>true</c> if the <see cref="FramePerSecond"/> and <see cref="TimePerFrame"/> were updated for this frame; otherwise, <c>false</c>.</value>
         public bool FramePerSecondUpdated { get; private set; }
 
+
+
+        /// <summary>
+        /// Gets the amount of time elapsed multiplied by the time factor.
+        /// </summary>
+        /// <value>The warped elapsed time</value>
+        public TimeSpan WarpElapsed 
+        {
+            // TODO: When switching with .NET 5, use the multiply operator instead of this. elapsedGameTime * Factor
+            get => TimeSpan.FromSeconds(Elapsed.TotalSeconds * Factor);
+        }
+
+
+        /// <summary>
+        /// Gets or sets the time factor for the time factor.
+        /// </summary>
+        /// <value>The multiply factor, a double between 0 and 1.</value>
```

Comment:
this should be changed, maybe "The multiply factor, higher or equal to 0"

Reply:
Ah right! i forgot the documentation!

Reply:
Did the docs receive an update?